tests/test_overrides_parser.py

```diff
@@ -1812,3 +1816,95 @@ def test_glob_filter(
 ) -> None:
     strings = ["the", "quick", "brown", "fox", "jumped", "under", "the", "lazy", "dog"]
     assert Glob(include=include, exclude=exclude).filter(strings) == expected
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "prefix,override_type",
+    [
+        pytest.param("", OverrideType.CHANGE, id="change"),
+        pytest.param("+", OverrideType.ADD, id="add"),
+        pytest.param("~", OverrideType.DEL, id="del"),
```

Comment:
we don't really need to rest everything in those 3 prefixes.
I would add explicit testing for + and ~ though, with and without a space between them and the key.

Reply:
> I would add explicit testing for + and ~ though, with and without a space between them and the key.

Just to be sure, do you mean tests verifying that with a space there is a crash? (whitespaces aren't allowed anymore in KEY mode due to 977bca69471a4e1229479ca412bc65d74bb58429)

Reply:
This point hasn't been resolved yet -- I wasn't sure exactly what you meant

Reply:
Missed your question.
I mean that I think we don't need dimension of "prefix,override_type", we can just drop it and hard code for normal override.
This is tripling the number of tests here and I don't think it's relevant for what you are actually testing.

Reply:
Got it, thanks! Done and rebased (now in e35815bc)